spanner/setup.py

```diff
@@ -30,7 +30,7 @@
 release_status = 'Development Status :: 5 - Production/Stable'
 dependencies = [
     'google-cloud-core<0.29dev,>=0.28.0',
-    'google-api-core[grpc]<2.0.0dev,>=1.1.0',
+    'google-api-core[grpc, grpcio-gcp]<2.0.0dev,>=1.1.0',
```

Comment:
We'll need to raise the minimum version here to the *next* release of api-core.

(Also this is the first time I've seen multiple extras used, does this even work?)

Reply:
Let's go ahead and update it in this PR.

Reply:
The first example in the [PEP 508 specification section](https://www.python.org/dev/peps/pep-0508/#specification) shows multiple extras:

```
requests [security,tests] >= 2.8.1, == 2.8.* ; python_version < "2.7"
```

pip doesn't complain, but it may not do the right thing either (as with other extras).

Reply:
ok. Just to make sure, the next release of api-core should be 1.3.1 , correct?

Reply:
Let's do 1.4.0.

Reply:
> Let's do 1.4.0

Right.  The new extra is definitely a user-visible "feature" of `api_core`:  they won't be able to install `google-cloud-apicore[grpcio-gcp]` while downgrading.